src/tf_container/serve.py

```diff
@@ -75,11 +75,25 @@ def export_saved_model(checkpoint_dir, model_path, s3=boto3.client('s3')):
             s3.download_file(bucket_name, key, target)
     else:
         if os.path.exists(checkpoint_dir):
-            shutil.copy2(checkpoint_dir, model_path)
+            _recursive_copy(checkpoint_dir, model_path)
```

Comment:
might be simpler and safer to `shutil.rmtree` whatever is in model_path and use [`shutil.copytree`](https://docs.python.org/3/library/shutil.html#shutil.copytree) to do the recursive copy, unless you want the union behavior of `_recursive_copy`, which I don't think you do.